src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,7 +55,7 @@
  */
 public class RobotContainer {
     // Subsystems
-    private final BaseDrivetrain driveSubsystem;
+    private final BaseSwerveSubsystem driveSubsystem;
```

Comment:
Why's this necessary?

Reply:
Even if the balancer command *required* a swerve subsystem, you could just do
```java
if (driveSubsystem instanceof BaseSwerveSubsystem) {
    // ...
    driveController.getBalancerButton().whileTrue(new BalancerCommand(swerveSubsystem));
}
```
in the `configureButtonBindings` `BaseSwerveSubsystem` if case.